test/README.md

```diff
@@ -21,8 +21,32 @@ Some examples of test harnesses using these tests, from the source repository:
 - [Formatting tests](https://github.com/messageformat/messageformat/blob/11c95dab2b25db8454e49ff4daadb817e1d5b770/packages/mf2-messageformat/src/messageformat.test.ts)
 
 A [JSON schema](./schemas/) is included for the test files in this repository.
-
-For users of Visual Studio Code, a [settings file](./.vscode/settings.json) is included that enables schema validation while editing the test files.
+## Error Codes
+
+The following table relates the error names used in the [JSON schema](./schemas/)
+to the error names used in ["MessageFormat 2.0 Errors"](../spec/errors.md) in the spec.
```

Comment:
Consider reversing the table? You're thinking about the errors from the point of view of the schema, but users of the spec will come here to find the ID

Reply:
Done in [c609a2f](https://github.com/unicode-org/message-format-wg/pull/850/commits/c609a2fc88c0c2bc2f5be415675ca06a3d84ec68)